### PR TITLE
Add 'decodeFunctionArgs' to decode function type args

### PR DESCRIPTION
**Pull request by xuewuli**
Add 'decodeFuncionArgs' to decode function type args & some typo fix

### CODE REVIEW

**Comment by xuewuli** [comment]
> do you want to help me?

yes, i think,we can fill a issue

File: index.js [hunk 1]
```diff
@@ -63,7 +63,41 @@ function encodeConstructorArgs(inputs) {
...
 }
 
+/**
+ * Decodes function args.
+ *
+ * @param {Object} contractABI - ABI of contract whose args to decode
+ * @param {string} bytecode - full call args bytecode
+ * @returns {Object} decodedArgs - Object representing decoded args with name, type, and data fields
+ */
+function decodeFuncionArgs(contractABI, bytecode) {
```
Comment (dmdque) [inline index.js]: Could you fix the typos? (Missing a "t")

Comment (xuewuli) [reply index.js]: my bad, fixed.

**Comment by dmdque** [comment]
Thanks for your contribution!

**Review by Sdsmetamask — COMMENTED** [review]
Merge

**Pull request merged**